docs/en/query_language/select.md

```diff
@@ -332,42 +334,53 @@ The query can only specify a single ARRAY JOIN clause.
 
-The table name can be specified instead of a subquery. This is equivalent to the `SELECT * FROM table` subquery, except in a special case when the table has the Join engine – an array prepared for joining.
+**Supported types of `JOIN`**
 
-All columns that are not needed for the JOIN are deleted from the subquery.
+- `INNER JOIN`
+- `LEFT OUTER JOIN`
+- `RIGHT OUTER JOIN`
+- `FULL OUTER JOIN`
+- `CROSS JOIN`
 
-There are several types of JOINs:
+**`ANY` or `ALL` strictness**
 
-`INNER` or `LEFT` type:If INNER is specified, the result will contain only those rows that have a matching row in the right table.
-If LEFT is specified, any rows in the left table that don't have matching rows in the right table will be assigned the default value - zeros or empty rows. LEFT OUTER may be written instead of LEFT; the word OUTER does not affect anything.
+If `ALL` is specified and the right table has several matching rows, the data will be multiplied by the number of these rows. It is a normal `JOIN` behavior from standard SQL.
+If `ANY` is specified and the right table has several matching rows, only the first one found is joined. If the right table has only one matching row, the results of `ANY` and `ALL` are the same.
 
-`ANY` or `ALL` stringency:If `ANY` is specified and the right table has several matching rows, only the first one found is joined.
-If `ALL` is specified and the right table has several matching rows, the data will be multiplied by the number of these rows.
+You can set the default value of strictness with session configuration parameter [join_default_strictness](../operations/settings/settings.md#session-setting-join_default_strictness).
 
-Using ALL corresponds to the normal JOIN semantic from standard SQL.
-Using ANY is optimal. If the right table has only one matching row, the results of ANY and ALL are the same. You must specify either ANY or ALL (neither of them is selected by default).
+**`GLOBAL` distribution**
 
-`GLOBAL` distribution:
-
-When using a normal JOIN, the query is sent to remote servers. Subqueries are run on each of them in order to make the right table, and the join is performed with this table. In other words, the right table is formed on each server separately.
+When using a normal `JOIN`, the query is sent to remote servers. Subqueries are run on each of them in order to make the right table, and the join is performed with this table. In other words, the right table is formed on each server separately.
 
 When using `GLOBAL ... JOIN`, first the requestor server runs a subquery to calculate the right table. This temporary table is passed to each remote server, and queries are run on them using the temporary data that was transmitted.
 
-Be careful when using GLOBAL JOINs. For more information, see the section "Distributed subqueries".
+Be careful when using `GLOBAL`. For more information, see the section [Distributed subqueries](#queries-distributed-subquries).
+
+**Peculiar properties of use**
```

Comment:
* Sounds weird, I'd call it something like "Implementation details"
* Also, why those are just bold instead of `####`?

Reply:
It is not "Implementation details", it is nearer to "Usage Recommendations".
I have used bold because I do not want to overload the right-side table of contents.